now.json

```diff
@@ -11,6 +11,7 @@
   "env": {
     "GOOGLE_SERVICE": "@google-service",
     "GOOGLE_OAUTH": "@google-oauth",
-    "FAUNA_SERVER_KEY": "@fauna-server-key"
+    "FAUNA_SERVER_KEY": "@fauna-server-key",
+    "FAUNADB_SECRET_KEY": "@faunadb_secret_key"
```

Comment:
the addition of the second entry seems to be a duplicate of the first?

Reply:
Eventually we might need two keys with different levels of access permissions to Fauna.
1. Just has read permission so that events can be displayed.
2. Has write permission (only used for creating new events).

But right now we have only one key to do both so for now it is a mistake, I will remove it.